File: gescompte_uml/src/views/CompteView.java
package views;
import java.util.ArrayList;
import entity.Compte;


public class CompteView extends View {
    
    public CompteView(){

       }
      public   Compte saisieCompte()  {
        Compte compte =new Compte();
        String num;
        compte.setSolde(sasieDouble("\"Entrer le solde du compte\""));
        return compte;

    } 

    public  void afficheCompte(ArrayList<Compte> comptes ){
        for (int index = 0; index < comptes.size(); index++) {
            System.out.println(comptes.get(index).toString());
        }
    }
}

File: php_gestion_compte/src/views/CompteView.php
<?php 
require_once "./src/models/Compte.php";
class CompteView{
      public static function saisieCompte():Compte{
         $compte=new Compte();
         do {
            $solde =(float)readline("Veuillez entrer le solde");
         } while ($solde <= 0);
         $compte->setSolde( $solde);
         return  $compte;
      }

      public static function afficheCompte(array $comptes):void{
        foreach ($comptes as $value) {
               print $value;
        }
      }
}
